cloud-controller-manager/civo/cloud.go

```diff
@@ -8,13 +8,11 @@ import (
 	cloudprovider "k8s.io/cloud-provider"
 )
 
-const (
-	// ProviderName is the name of the provider.
-	ProviderName string = "civo"
+// ProviderName is the name of the provider.
+const ProviderName string = "civo"
 
-	// CCMVersion is the version of the CCM.
-	CCMVersion string = "dev"
-)
+// CCMVersion is the version of the CCM.
+var CCMVersion string = "dev"
```

Comment:
>[!NOTE]
This change is related to the following issue. https://github.com/civo/civo-cloud-controller-manager/issues/9
Although the information was intended to be written at build time, the variable was declared as a constant, so the value was not overwritten and remained as "dev".
Ref: https://github.com/civo/civo-cloud-controller-manager/blob/master/Dockerfile#L8
```
RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -ldflags "-s -w -X github.com/civo/civo-cloud-controller-manager/cloud-controller-manager/civo.CCMVersion=${VERSION}" -o civo-cloud-controller-manager github.com/civo/civo-cloud-controller-manager/cloud-controller-manager/cmd/civo-cloud-controller-manager
```